Cargo.toml

```diff
@@ -15,6 +15,9 @@ opt-level = 3
 # lto = false
 
 [patch.crates-io]
+# FIXME: https://github.com/servo/servo/issues/18044
+xcb = {git = "https://github.com/servo/rust-xcb", rev = "servo_issue18044_do_not_remove"}
```

Comment:
This `Cargo.toml` is not "top-level" in Gecko, so that section is ignored. Which doesn’t matter because xcb is not in Gecko’s dependency graph anyway.